applications/tari_app_grpc/proto/types.proto

```diff
@@ -67,12 +67,12 @@ message ProofOfWork {
 }
 
 //This is used to request the which pow algo should be used with the block template
-message PowAlgo{
-    enum PowAlgo {
+message PowAlgo {
+    enum PowAlgos {
         POW_ALGO_MONERO = 0;
         POW_ALGO_BLAKE = 1;
```

Comment:
Prefix each variant with the name of the enum - this way the generated rust enum will be `PowAlgos::Monero` instead of `PowAlgos::PowAlgoMonero`

```suggestion
        POW_ALGOS_MONERO = 0;
        POW_ALGOS_BLAKE = 1;
```

Reply:
Thanks @sdbondi